mozilla-mobile/android-components/components/compose/browser-toolbar/src/main/res/values/strings.xml

```diff
@@ -4,4 +4,10 @@
     <string name="mozac_clear_button_description">Clear</string>
     <!-- Placeholder text shown in the search bar before a user enters text for the default engine -->
     <string name="mozac_browser_toolbar_search_hint">Search or enter address</string>
+    <!-- Contextual menu option for long pressing on the toolbar URL. Clicking it will copy the current URL -->
+    <string name="mozac_browser_toolbar_long_press_popup_copy">Copy</string>
+    <!-- Contextual menu option for long pressing on the toolbar URL. Clicking it will paste the text from clipboard and allow user to edit it -->
+    <string name="mozac_browser_toolbar_long_press_popup_paste">Paste</string>
+    <!-- Contextual menu option for long pressing on the toolbar URL. Clicking it will paste the text from clipboard and search for or load it as an URL. '&amp;' is replaced with the ampersand symbol: & -->
```

Comment:
Pretty sure this will show up as just `&` in Pontoon, so the comment might be more confusing than helpful. But checking in a few minutes…

Reply:
@Delphine 

I was correct. Mostly an FYI for the future.

<img width="812" alt="Screenshot 2025-05-15 alle 07 16 40" src="https://github.com/user-attachments/assets/bed4d8a4-33c2-4670-883f-d9c688234f4b" />